### ISSUE TITLE
Organic Chemistry - Webview - Chemistry Matters section not appearing correctly in the Webview Table of Contents

### ISSUE THREAD

**Issue by ww-openstax** [openstax/cnx-recipes#5182]
# Brief Description of Scenario
Chemistry Matters section is not appearing correctly in the Webview Table of Contents
Sometimes "Chemistry Matters" appears in the Webview TOC and sometimes it doesn't. Sometimes "Chemistry Matters" is partially appearing and other times it is blocked by other content.
It also seems like full-justification is set for the Chem Matters boxes in the Webview TOC but no other Webview TOC elements seem to have that justification.
image.png
 
![image](https://github.com/openstax/cnx-recipes/assets/89537088/670049f7-9d3f-4fe5-811b-bdd1153e8216)

# Pattern of Occurrence 
All Chemistry Matters sections

# Acceptance Criteria

  
# Sample Mark-Up
```
<!-- Please paste here if applicable. -->
```

# Location(s)
- **osbooks repo:** osbooks-organic-chemistry
- **collection slug:** organic-chemistry
- **ref/branch:**
- **chapter/section:** all Chemistry Matters sections
- **surrounding text:**

**Comment by kswanson33** [comment]
compare the html structure of `chemistry matters` with `key terms`:

![image.png](https://images.zenhubusercontent.com/5fa9a1a6a99686111a824015/22accc49-c6f2-4066-b358-2eb246a28377)

^ key terms h2 only contains a `<span>` with the title

![image.png](https://images.zenhubusercontent.com/5fa9a1a6a99686111a824015/10f1bae7-3163-4153-91e7-571917d45ce6)

^ chemistry matters h2 contains several elements


In the web TOC, all the elements within the h2 are being brought in. here it is full screen:

![image.png](https://images.zenhubusercontent.com/5fa9a1a6a99686111a824015/ff203694-a3ce-4a3c-a015-4a6dc27959ff)

What's happening is, with multiple elements in this title, the flexbox styling is causing "Chemistry Matters - <subtitle>" to display the same way as numbered section pages (i.e. "<number> <divider> <title>").

Questions:

- Do we want both "Chemistry Matters" and subtitle to display in the TOC? Or just one?
- Fix in CSS or HTML? If CSS, this is almost certainly a rex team fix, since the web styles only cover what's within a page, not the TOC. If HTML, this is a styles team fix.

**Comment by lwtchu** [comment]
Decision: make the subtitle a sibling for future work but for now we can keep the web-specific TOC to say Chemistry Matters

Acceptance criteria
- web toc should say Chemistry Matters
- PDF should appear the same

**Comment by lwtchu** [comment]
@kswanson33 we'll need to suppress the 1 Chemistry Matters injected text right? in that last screenshot

**Comment by kswanson33** [comment]
yes ... i think the text is not being suppressed because of [something new](https://github.com/openstax/ce-styles/blob/main/styles/books/organic-chemistry/book.scss#L14) going on with folio for ochem. this would be a different card

**Comment by kswanson33** [comment]
for testing: check for regressions in PDF & fix on web. if using rex-preview, note [new docs](https://github.com/openstax/enki/pull/281/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R84-R88)

**Comment by Stackblocks** [comment]
Full title now showing properly in the web view at a variety of page and sidebar widths, checking for PDF regressions next.

![CleanShot 2023-07-24 at 09.55.12.png](https://images.zenhubusercontent.com/5af1f4c24b5806bc2bd257f2/74b1002c-9906-4a8a-b45f-a5349f125c2a)

![CleanShot 2023-07-24 at 09.57.40.png](https://images.zenhubusercontent.com/5af1f4c24b5806bc2bd257f2/efa49c2b-22ff-4e61-9aac-3446c0dd56cd)
